lib/pages/chat/chat.dart

```diff
@@ -116,23 +117,25 @@ class ChatController extends State<Chat>
       );
 
   final AutoScrollController scrollController = AutoScrollController();
-  final AutoScrollController forwardListController = AutoScrollController();
+  final KeyboardVisibilityController keyboardVisibilityController =
+      KeyboardVisibilityController();
   final ValueNotifier<String?> focusHover = ValueNotifier(null);
   final ValueNotifier<bool> openingPopupMenu = ValueNotifier(false);
-
+  final ValueNotifier<bool> draggingNotifier = ValueNotifier(false);
+  final ValueNotifier<bool> scrollDownButtonNotifier = ValueNotifier(false);
```

Comment:
showScrollDownButtonNotifier

Reply:
@nqhhdev

Reply:
No show, I think it `activeDownButtonNotifierr`